Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -1142,6 +1142,9 @@ private extension PurchasesOrchestrator {
                     self.backend.post(receipt: .empty,
                                       productData: nil,
                                       transactionData: .init(appUserID: currentAppUserID,
+                                                             presentedOfferingContext: nil,
+                                                             unsyncedAttributes: unsyncedAttributes,
+                                                             storefront: await Storefront.currentStorefront,
```

Comment:
we dont really need the storefront here I think since there are no purchases

Reply:
nitpick: maybe it's worth splitting this into its own line, like we do in the else code path, that explicitly does `createProductRequestData`

Reply:
@aboedo Done! We're now creating `transactionData` on it's own line 😎